src/components/labelcontainer.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import err                  from '../err';
+
+class LabelContainer extends Component{
+    constructor(props){
+        super(props);
+
+        // Defining Component Requirements
+        const requiredKeys = [
+            'id',
+            'style',
+            'onClick',
+            'errorLine',
+            'lines',
+            'colors',
+            'reference'
+        ];
+        err.missingAnyKeys('this.props',this.props,requiredKeys);
+        err.missingAnyKeys('this.props.style',this.props.style,['container','labels']);
+        this.renderLabels = this.renderLabels.bind(this);
+    }
+    renderLabels(){
+        const { errorLine, lines, colors, style } = this.props;
+        let
+            labels    = new Array(lines);
+        for(var i = 0; i < lines - 1; i++) labels[i] = i + 1;
```

Comment:
I think we should set a rule not to use var in this project. Please tell me how you think about this.
Most linters and JavaScript guidelines agree that using var causes more trouble than it is worth. In this scenario I see no benefit of using `var` over `let`.